Index.js

```diff
@@ -0,0 +1,71 @@
+// eslint-disable-next-line max-classes-per-file
+
+import Books from './Modules/Bookslist.js';
+import timenow from './Modules/Date.js';
+
+const bookstore = new Books();
+
+if (localStorage.getItem('books')) {
+  bookstore.printfn();
+}
+
+const addBtn = document.getElementById('btn');
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+
+addBtn.addEventListener('click', () => {
+  if (title.value !== '' && author.value !== '') {
+    bookstore.addbooks(title.value, author.value);
+    document.forms[0].reset();
+  }
+});
+
+document.addEventListener('click', (removed) => {
+  const { id } = removed.target;
+  if (id.substr(0, 4) === 'book') {
+    bookstore.deletebook(id);
+  }
+});
+document.getElementById('lis').classList.add('active');
+
+document.querySelector('.contact').style.display = 'none';
+document.querySelector('.book').style.display = 'block';
+document.querySelector('.add').style.display = 'none';
+
+document.querySelector('#con').addEventListener('click', () => {
+  document.getElementById('con').classList.add('active');
+  document.getElementById('lis').classList.remove('active');
+  document.getElementById('new').classList.remove('active');
+  document.querySelector('.contact').style.display = 'block';
+  document.querySelector('.book').style.display = 'none';
+  document.querySelector('.add').style.display = 'none';
+});
+
+document.querySelector('#new').addEventListener('click', () => {
+  document.getElementById('new').classList.add('active');
+  document.getElementById('con').classList.remove('active');
+  document.getElementById('lis').classList.remove('active');
+  document.querySelector('.contact').style.display = 'none';
+  document.querySelector('.book').style.display = 'none';
+  document.querySelector('.add').style.display = 'block';
+});
+
+document.querySelector('#lis').addEventListener('click', () => {
+  document.getElementById('lis').classList.add('active');
+  document.getElementById('con').classList.remove('active');
+  document.getElementById('new').classList.remove('active');
+  document.querySelector('.contact').style.display = 'none';
+  document.querySelector('.book').style.display = 'block';
+  document.querySelector('.add').style.display = 'none';
+});
+
+document.querySelector('#logo').addEventListener('click', () => {
+  document.querySelector('.contact').style.display = 'none';
+  document.querySelector('.book').style.display = 'block';
+  document.querySelector('.add').style.display = 'none';
+});
```

Comment:
- [OPTIONAL] Great job here 💯, but I think this code could be improved because I noticed you repeated the same code many times (DRY)
> You can check this for more info [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

> I would suggest creating a single `gotTo()` function and using that instead
> For Example 💡
```js
const goTo = (target) => {
  document.getElementById('new').classList.remove('active');
  document.getElementById('con').classList.remove('active');
  document.getElementById('lis').classList.add('active');
  if (target === 'contact') {
    document.querySelector('.contact').style.display = 'block';
    document.querySelector('.book').style.display = 'none';
    document.querySelector('.add').style.display = 'none';
  } else if (target === 'list') {
    document.querySelector('.contact').style.display = 'none';
    document.querySelector('.book').style.display = 'block';
    document.querySelector('.add').style.display = 'none';
  } else {
    document.querySelector('.contact').style.display = 'none';
    document.querySelector('.book').style.display = 'none';
    document.querySelector('.add').style.display = 'block';
  }
};

document.querySelector('#con').addEventListener('click', () => goTo('contact'));
document.querySelector('#new').addEventListener('click', () => goTo('new'));
document.querySelector('#lis').addEventListener('click', () => goTo('list'));
document.querySelector('#logo').addEventListener('click', () => goTo('list'));
```